.buildkite/test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -ex
+
```

Comment:
Don’t need this. The CI does it as part of pulling the repo

Reply:
Hmm I removed this and don't see the steps in the output anymore. Going to re-add

Reply:
lol meant the submodule recurse init thing, all good.